app-modules/service-management/database/seeders/ChangeRequestTypeSeeder.php

```diff
@@ -36,20 +36,28 @@
 
 namespace AdvisingApp\ServiceManagement\Database\Seeders;
 
+use App\Models\User;
 use Illuminate\Database\Seeder;
 use AdvisingApp\ServiceManagement\Models\ChangeRequestType;
 
 class ChangeRequestTypeSeeder extends Seeder
 {
     public function run(): void
     {
-        $types = ['Standard Change', 'Normal Change', 'Emergency Change'];
+        $types = [
+            ['Normal Change', 0],
+            ['Standard Change', 1],
+            ['Emergency Change', 2],
+        ];
 
         foreach ($types as $type) {
-            ChangeRequestType::factory()
+            $changeRequestType = ChangeRequestType::factory()
                 ->create([
-                    'name' => $type,
+                    'name' => $type[0],
+                    'number_of_required_approvals' => $type[1],
                 ]);
+
+            $changeRequestType->userApprovers()->attach(User::where('email', config('local_development.super_admin.email'))->first()->id);
```

Comment:
Let's surround this in a check as to whether or not we are in Production. That way we can use this seeder to seed the default Change Request Types that come with a new installation.